internal/service/metricservice/service.go

```diff
@@ -0,0 +1,13 @@
+package metricservice
+
+import "github.com/VadimOcLock/metrics-service/internal/store/somestore"
+
+type Service struct {
+	Store somestore.Store
+}
+
+func New(s somestore.Store) Service {
```

Comment:
Я бы рекомендовал возвращать указатель вместо значения. Это позволит избежать лишнего копирования, а также позволит трактовать `nil` более однозначно, как отсутствие значения.
```suggestion
func New(s somestore.Store) *Service {
```